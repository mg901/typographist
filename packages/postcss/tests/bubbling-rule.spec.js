const run = require('./run');

test('transform bubbling rule', async () => {
  const source = `
        :global {
          .test {
            color: red;
          }
          .some-item {
            font-size: 1rem;
          }
        }
        :local {
          .item {
            width: 200px;
          }
        }`;

  expect(await run(source)).toMatchSnapshot();
});
