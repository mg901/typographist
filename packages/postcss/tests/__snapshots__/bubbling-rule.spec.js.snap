// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform bubbling rule 1`] = `
:global .test {
  color: red;
}
:global .some-item {
  font-size: 1rem;
}
:local .item {
  width: 200px;
}
`;
